activerecord/test/cases/finder_test.rb

```diff
@@ -797,10 +798,40 @@ def test_implicit_order_column_is_configurable
 
     assert_equal topics(:fifth), Topic.first
     assert_equal topics(:third), Topic.last
+
+    c = Topic.connection
+    table_name = c.quote_table_name("topics")
+    assert_sql(/ORDER BY #{Regexp.escape(table_name)}.#{Regexp.escape(c.quote_column_name("title"))} DESC, #{Regexp.escape(table_name)}.#{Regexp.escape(c.quote_column_name("id"))} DESC LIMIT/i) {
```

Comment:
These lines could be shorter and more readable if the table names and column names were quoted together:

```suggestion
    assert_sql(/ORDER BY #{Regexp.escape(c.quote_table_name("topics.title"))} DESC, #{Regexp.escape(c.quote_table_name("topics.id"))} DESC LIMIT/i) {
```

It's admittedly a little strange to quote a column name with the `quote_table_name` method, but it works, and we already rely on it elsewhere:

https://github.com/rails/rails/blob/98754de1412870d7dae9eba1c7bac944b9b90093/activerecord/test/cases/batches_test.rb#L149
https://github.com/rails/rails/blob/98754de1412870d7dae9eba1c7bac944b9b90093/activerecord/test/cases/calculations_test.rb#L143